modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -320,4 +326,8 @@ export class StakingWallet implements IStakingWallet {
       return true;
     }
   }
+
+  private async getDescriptorWallet(transaction: StakingTransaction): Promise<IWallet> {
+    return await this.wallet.baseCoin.wallets().get({ id: transaction.buildParams?.senderWalletId });
+  }
```

Comment:
Im confused on why `senderWalletId` is passed in? 

How is `senderWalletId` different then `wallet._id` ?

Reply:
it is when the sender wallet is different from wallet._id, it only happens for btc unstaking because btc created a separate descriptor wallet